packages/bot-web-ui/src/stores/app-store.js

```diff
@@ -141,7 +141,12 @@ export default class AppStore {
         this.disposeLandingCompanyChangeReaction = reaction(
             () => client.landing_company_shortcode,
             () => {
-                if (client.has_maltainvest_account || client.is_options_blocked) {
+                if (
+                    (!client.is_logged_in && client.is_eu_country) ||
```

Comment:
I think this `is_eu_country)` is based on IP so no need to check the login status.

Reply:
@sara-fs The condition is `EU IP while not logged in`
The conditions here check for an EU IP which is not logged in. Right?